contracts/REXUniswapV3Market.sol

```diff
@@ -450,7 +450,12 @@ contract REXUniswapV3Market is
 
         // Downgrade if this is not a supertoken
         if (underlyingInputToken != address(inputToken)) {
-            inputToken.downgrade(inputToken.balanceOf(address(this)));
+            try inputToken.downgrade(inputToken.balanceOf(address(this))) {} catch {
+                
+                ISETHCustom(address(inputToken)).downgradeToETH(
+                    inputToken.balanceOf(address(this))
+                );
+            }
```

Comment:
LGTM, this is how I had it in mind, `downgrade` is the common case so try that first, if that method doesn't exist use `downgradeToETH`